system/include/emscripten/em_asm.h

```diff
@@ -1,35 +1,135 @@
 #ifndef __em_asm_h__
 #define __em_asm_h__
 
-// The wasm backend calls vararg functions by passing the variadic arguments as
-// an array on the stack. For EM_ASM's underlying functions, s2wasm needs to
-// translate them to emscripten_asm_const_[signature], but the expected
-// signature and arguments has been lost as part of the vararg buffer.
-// Therefore, we declare EM_ASM's implementing functions as non-variadic.
-
 #ifndef __asmjs
```

Comment:
i don't remember where we define this. is it not `__asmjs__`?

Reply:
It looks like we define both (https://github.com/kripken/emscripten/blob/incoming/site/source/docs/compiling/Building-Projects.rst#detecting-emscripten-in-preprocessor), but grepping both emscripten and fastcomp doesn't show anything obvious for where they get defined. This also seems to be the only use of `__asmjs` instead of `__asmjs__`, so I'll make it consistent